### DISCUSSION TITLE
可以提供支持Mac Catalyst架构的xcframework吗

### DISCUSSION THREAD

**Discussion by Rogue24** [Tencent/libpag#2433] [General] [answered]
目前masOS和iOS架构无法在Mac Catalyst中使用

**Comment by kevingpqi123** [comment, marked as answer]
由于 PAG 目前 GPU 渲染基于的是 OpenGL ES，而 Mac Catalyst 并不支持 OpenGL ES，因此需要等 PAG 支持 Metal 之后才能支持 Mac Catalyst， 短期内还无法提供

**Reply by Rogue24** [reply]
好吧 谢谢
